deepparse/dataset_container/dataset_container.py

```diff
@@ -0,0 +1,47 @@
+from abc import ABC, abstractmethod
+from pickle import load
+from typing import Union
+
+from torch.utils.data import Dataset
+
+
+class DatasetContainerInterface(Dataset, ABC):
```

Comment:
```suggestion
class DatasetContainer(Dataset, ABC):
```

Reply:
Better to change the class signature with an IDE so as not to break things

Reply:
f53dded